db/seeds.rb

```diff
@@ -42,9 +42,9 @@ def generate_users
   end
 
   def generate_dogs
-    100.times do |i|
+    99.times do |i|
       dog = Dog.create!(
-      user_id: i ,
+      user_id: i + 1 ,
```

Comment:
Indent the first parameter one step more than the previous line.<br>Space found before comma.